<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.549">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>The Linear Mixed Modelers - Literature</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
<meta property="og:title" content="The Linear Mixed Modelers - Literature">
<meta property="og:site_name" content="The Linear Mixed Modelers">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">The Linear Mixed Modelers</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./literature.html" aria-current="page"> 
<span class="menu-text">Literature</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./report.html"> 
<span class="menu-text">Report</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./slides.html"> 
<span class="menu-text">Slides</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./code.html"> 
<span class="menu-text">Code</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./about.html"> 
<span class="menu-text">About Us</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#joshs-literature-review" id="toc-joshs-literature-review" class="nav-link active" data-scroll-target="#joshs-literature-review">Josh’s Literature Review</a>
  <ul class="collapse">
  <li><a href="#fitting-linear-mixed-effects-models-using-lme4-journal-of-statistical-software-jstatsoft.org" id="toc-fitting-linear-mixed-effects-models-using-lme4-journal-of-statistical-software-jstatsoft.org" class="nav-link" data-scroll-target="#fitting-linear-mixed-effects-models-using-lme4-journal-of-statistical-software-jstatsoft.org">Fitting Linear Mixed-Effects Models Using lme4 | Journal of Statistical Software (jstatsoft.org)</a></li>
  <li><a href="#statistical-primer-an-introduction-to-the-application-of-linear-mixed-effects-models-in-cardiothoracic-surgery-outcomes-research-a-case-study-using-homograft-pulmonary-valve-replacement-data---pubmed-nih.gov" id="toc-statistical-primer-an-introduction-to-the-application-of-linear-mixed-effects-models-in-cardiothoracic-surgery-outcomes-research-a-case-study-using-homograft-pulmonary-valve-replacement-data---pubmed-nih.gov" class="nav-link" data-scroll-target="#statistical-primer-an-introduction-to-the-application-of-linear-mixed-effects-models-in-cardiothoracic-surgery-outcomes-research-a-case-study-using-homograft-pulmonary-valve-replacement-data---pubmed-nih.gov">Statistical primer: an introduction to the application of linear mixed-effects models in cardiothoracic surgery outcomes research-a case study using homograft pulmonary valve replacement data - PubMed (nih.gov)</a></li>
  <li><a href="#generalized-linear-mixed-models-a-practical-guide-for-ecology-and-evolution---sciencedirect" id="toc-generalized-linear-mixed-models-a-practical-guide-for-ecology-and-evolution---sciencedirect" class="nav-link" data-scroll-target="#generalized-linear-mixed-models-a-practical-guide-for-ecology-and-evolution---sciencedirect">Generalized linear mixed models: a practical guide for ecology and evolution - ScienceDirect</a></li>
  <li><a href="#frontiers-linear-mixed-effects-models-for-within-participant-psychology-experiments-an-introductory-tutorial-and-free-graphical-user-interface-lmmgui-frontiersin.org" id="toc-frontiers-linear-mixed-effects-models-for-within-participant-psychology-experiments-an-introductory-tutorial-and-free-graphical-user-interface-lmmgui-frontiersin.org" class="nav-link" data-scroll-target="#frontiers-linear-mixed-effects-models-for-within-participant-psychology-experiments-an-introductory-tutorial-and-free-graphical-user-interface-lmmgui-frontiersin.org">Frontiers | Linear mixed-effects models for within-participant psychology experiments: an introductory tutorial and free, graphical user interface (LMMgui) (frontiersin.org)</a></li>
  <li><a href="#a-linear-mixed-model-to-estimate-covid-19-induced-excess-mortality---pubmed-nih.gov" id="toc-a-linear-mixed-model-to-estimate-covid-19-induced-excess-mortality---pubmed-nih.gov" class="nav-link" data-scroll-target="#a-linear-mixed-model-to-estimate-covid-19-induced-excess-mortality---pubmed-nih.gov">A linear mixed model to estimate COVID-19-induced excess mortality - PubMed (nih.gov)</a></li>
  <li><a href="#when-a-joint-model-should-be-preferred-over-a-linear-mixed-model-for-analysis-of-longitudinal-health-related-quality-of-life-data-in-cancer-clinical-trials---pubmed-nih.gov" id="toc-when-a-joint-model-should-be-preferred-over-a-linear-mixed-model-for-analysis-of-longitudinal-health-related-quality-of-life-data-in-cancer-clinical-trials---pubmed-nih.gov" class="nav-link" data-scroll-target="#when-a-joint-model-should-be-preferred-over-a-linear-mixed-model-for-analysis-of-longitudinal-health-related-quality-of-life-data-in-cancer-clinical-trials---pubmed-nih.gov">When a joint model should be preferred over a linear mixed model for analysis of longitudinal health-related quality of life data in cancer clinical trials - PubMed (nih.gov)</a></li>
  </ul></li>
  <li><a href="#jacobs-literature-review" id="toc-jacobs-literature-review" class="nav-link" data-scroll-target="#jacobs-literature-review">Jacob’s Literature Review</a>
  <ul class="collapse">
  <li><a href="#multilevel-analysis-quantifies-variation-in-the-experimental-effect-while-optimizing-power-and-preventing-false-positives" id="toc-multilevel-analysis-quantifies-variation-in-the-experimental-effect-while-optimizing-power-and-preventing-false-positives" class="nav-link" data-scroll-target="#multilevel-analysis-quantifies-variation-in-the-experimental-effect-while-optimizing-power-and-preventing-false-positives">Multilevel analysis quantifies variation in the experimental effect while optimizing power and preventing false positives</a></li>
  <li><a href="#multimodel-inference-in-ecology-and-evolution-challenges-and-solutionsnih.gov" id="toc-multimodel-inference-in-ecology-and-evolution-challenges-and-solutionsnih.gov" class="nav-link" data-scroll-target="#multimodel-inference-in-ecology-and-evolution-challenges-and-solutionsnih.gov">Multimodel inference in ecology and evolution: challenges and solutions(nih.gov)</a></li>
  <li><a href="#linear-mixed-model-for-analyzing-longitudinal-data-a-simulation-study-of-children-growth-differences-sciencedirect.com" id="toc-linear-mixed-model-for-analyzing-longitudinal-data-a-simulation-study-of-children-growth-differences-sciencedirect.com" class="nav-link" data-scroll-target="#linear-mixed-model-for-analyzing-longitudinal-data-a-simulation-study-of-children-growth-differences-sciencedirect.com">Linear Mixed Model for Analyzing Longitudinal Data: A Simulation Study of Children Growth Differences (sciencedirect.com)</a></li>
  <li><a href="#a-protocol-for-conducting-and-presenting-results-of-regression-type-analyses" id="toc-a-protocol-for-conducting-and-presenting-results-of-regression-type-analyses" class="nav-link" data-scroll-target="#a-protocol-for-conducting-and-presenting-results-of-regression-type-analyses">A protocol for conducting and presenting results of regression-type analyses</a></li>
  <li><a href="#level-logical-explanations-visualizations-of-estimates-in-linear-mixed-models-recommendations-for-reporting-multilevel-data-and-analyses" id="toc-level-logical-explanations-visualizations-of-estimates-in-linear-mixed-models-recommendations-for-reporting-multilevel-data-and-analyses" class="nav-link" data-scroll-target="#level-logical-explanations-visualizations-of-estimates-in-linear-mixed-models-recommendations-for-reporting-multilevel-data-and-analyses">LEVEL (Logical Explanations &amp; Visualizations of Estimates in Linear mixed models): recommendations for reporting multilevel data and analyses</a></li>
  <li><a href="#estimation-and-selection-in-linear-mixed-models-with-missing-data-under-compound-symmetric-structure-nih.gov" id="toc-estimation-and-selection-in-linear-mixed-models-with-missing-data-under-compound-symmetric-structure-nih.gov" class="nav-link" data-scroll-target="#estimation-and-selection-in-linear-mixed-models-with-missing-data-under-compound-symmetric-structure-nih.gov">Estimation and selection in linear mixed models with missing data under compound symmetric structure (nih.gov)</a></li>
  </ul></li>
  <li><a href="#syeds-literature-review" id="toc-syeds-literature-review" class="nav-link" data-scroll-target="#syeds-literature-review">Syed’s Literature Review</a>
  <ul class="collapse">
  <li><a href="#an-introduction-to-linear-mixed-effects-modeling-in-r" id="toc-an-introduction-to-linear-mixed-effects-modeling-in-r" class="nav-link" data-scroll-target="#an-introduction-to-linear-mixed-effects-modeling-in-r">An Introduction to Linear Mixed-Effects Modeling in R</a></li>
  <li><a href="#a-brief-introduction-to-mixed-effects-modelling-and-multi-model-inference-in-ecology" id="toc-a-brief-introduction-to-mixed-effects-modelling-and-multi-model-inference-in-ecology" class="nav-link" data-scroll-target="#a-brief-introduction-to-mixed-effects-modelling-and-multi-model-inference-in-ecology">A brief introduction to mixed effects modelling and multi-model inference in ecology</a></li>
  <li><a href="#robustness-of-linear-mixed-effects-models-to-violations-of-distributional-assumptions" id="toc-robustness-of-linear-mixed-effects-models-to-violations-of-distributional-assumptions" class="nav-link" data-scroll-target="#robustness-of-linear-mixed-effects-models-to-violations-of-distributional-assumptions">Robustness of linear mixed-effects models to violations of distributional assumptions</a></li>
  <li><a href="#to-transform-or-not-to-transform-using-generalized-linear-mixed-models-to-analyse-reaction-time-data" id="toc-to-transform-or-not-to-transform-using-generalized-linear-mixed-models-to-analyse-reaction-time-data" class="nav-link" data-scroll-target="#to-transform-or-not-to-transform-using-generalized-linear-mixed-models-to-analyse-reaction-time-data">To transform or not to transform: using generalized linear mixed models to analyse reaction time data</a></li>
  <li><a href="#analysing-disease-incidence-data-from-designed-experiments-by-generalized-linear-mixed-models" id="toc-analysing-disease-incidence-data-from-designed-experiments-by-generalized-linear-mixed-models" class="nav-link" data-scroll-target="#analysing-disease-incidence-data-from-designed-experiments-by-generalized-linear-mixed-models">Analysing disease incidence data from designed experiments by generalized linear mixed models</a></li>
  </ul></li>
  <li><a href="#saras-literature-review" id="toc-saras-literature-review" class="nav-link" data-scroll-target="#saras-literature-review">Sara’s Literature Review</a>
  <ul class="collapse">
  <li><a href="#model-selection-in-linear-mixed-effect-models" id="toc-model-selection-in-linear-mixed-effect-models" class="nav-link" data-scroll-target="#model-selection-in-linear-mixed-effect-models">Model selection in linear mixed effect models</a></li>
  <li><a href="#random-effects-structure-for-testing-interactions-in-linear-mixed-effects-models" id="toc-random-effects-structure-for-testing-interactions-in-linear-mixed-effects-models" class="nav-link" data-scroll-target="#random-effects-structure-for-testing-interactions-in-linear-mixed-effects-models">Random Effects Structure for Testing Interactions in Linear Mixed-Effects Models</a></li>
  <li><a href="#pymer4-connecting-r-and-python-for-linear-mixed-modeling" id="toc-pymer4-connecting-r-and-python-for-linear-mixed-modeling" class="nav-link" data-scroll-target="#pymer4-connecting-r-and-python-for-linear-mixed-modeling">Pymer4: Connecting R and Python for Linear Mixed Modeling</a></li>
  <li><a href="#a-powerful-and-flexible-linear-mixed-model-framework-for-the-analysis-of-relative-quantification-rt-pcr-data" id="toc-a-powerful-and-flexible-linear-mixed-model-framework-for-the-analysis-of-relative-quantification-rt-pcr-data" class="nav-link" data-scroll-target="#a-powerful-and-flexible-linear-mixed-model-framework-for-the-analysis-of-relative-quantification-rt-pcr-data">A powerful and flexible linear mixed model framework for the analysis of relative quantification RT-PCR data</a></li>
  <li><a href="#using-generalized-linear-mixed-models-to-evaluate-inconsistency-within-a-network-meta-analysis" id="toc-using-generalized-linear-mixed-models-to-evaluate-inconsistency-within-a-network-meta-analysis" class="nav-link" data-scroll-target="#using-generalized-linear-mixed-models-to-evaluate-inconsistency-within-a-network-meta-analysis">Using Generalized Linear Mixed Models to Evaluate Inconsistency within a Network Meta-Analysis</a></li>
  <li><a href="#generalized-linear-mixed-model-glmm-trees-a-flexible-decision-tree-method-for-multilevel-and-longitudinal-data" id="toc-generalized-linear-mixed-model-glmm-trees-a-flexible-decision-tree-method-for-multilevel-and-longitudinal-data" class="nav-link" data-scroll-target="#generalized-linear-mixed-model-glmm-trees-a-flexible-decision-tree-method-for-multilevel-and-longitudinal-data">Generalized linear mixed-model (GLMM) trees: A flexible decision-tree method for multilevel and longitudinal data</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Literature</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="joshs-literature-review" class="level1">
<h1>Josh’s Literature Review</h1>
<section id="fitting-linear-mixed-effects-models-using-lme4-journal-of-statistical-software-jstatsoft.org" class="level2">
<h2 class="anchored" data-anchor-id="fitting-linear-mixed-effects-models-using-lme4-journal-of-statistical-software-jstatsoft.org"><a href="https://www.jstatsoft.org/article/view/v067i01">Fitting Linear Mixed-Effects Models Using lme4 | Journal of Statistical Software (jstatsoft.org)</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The paper presents the lme4 package for R, which facilitates the fitting of linear mixed-effects models. The authors aim to articulate the package’s capabilities in evaluating the profiled deviance or REML criterion for linear mixed models and to explain the representation and optimization of such models for parameter estimation.</p>
<p><strong>Why is it important?</strong></p>
<p>The significance of this paper lies in its contribution to the field of computational methods for fitting mixed models—an area with many open problems. The lme4 package represents an evolution in this domain, offering more efficient computational tools and a syntax that simplifies the modeling process, especially for models with crossed random effects.</p>
<p><strong>How is it solved? – methods</strong></p>
<p>The package utilizes maximum likelihood or restricted maximum likelihood (REML) estimates for linear mixed-effects model parameters, employing numerical representation and optimization functions within R. The paper delves into the model’s structure, the evaluative steps for the profiled deviance or REML criterion, and the class structure representing such models, highlighting the improvements over previous formulations.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>The document focuses more on methodology than specific results or limitations. It details the improvement over the nlme package, addressing efficient linear algebra tools and the incorporation of profile likelihood confidence intervals on random-effects parameters. The paper emphasizes the ongoing development of the lme4 package, acknowledging the need for stability and usability for a broad range of applications.</p>
</section>
<section id="statistical-primer-an-introduction-to-the-application-of-linear-mixed-effects-models-in-cardiothoracic-surgery-outcomes-research-a-case-study-using-homograft-pulmonary-valve-replacement-data---pubmed-nih.gov" class="level2">
<h2 class="anchored" data-anchor-id="statistical-primer-an-introduction-to-the-application-of-linear-mixed-effects-models-in-cardiothoracic-surgery-outcomes-research-a-case-study-using-homograft-pulmonary-valve-replacement-data---pubmed-nih.gov"><a href="https://pubmed.ncbi.nlm.nih.gov/36005884/">Statistical primer: an introduction to the application of linear mixed-effects models in cardiothoracic surgery outcomes research-a case study using homograft pulmonary valve replacement data - PubMed (nih.gov)</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The goal of the paper is to provide a detailed introduction to developing and interpreting linear mixed-effects models for repeated measurements in the context of cardiothoracic surgery outcomes research. The paper uses a dataset on patients undergoing surgical pulmonary valve replacement to illustrate the steps of developing such models for clinician researchers.</p>
<p><strong>Why is it important?</strong></p>
<p>This work is important because the emergence of large cardio-thoracic surgery datasets, including repeated measurements over time, presents an opportunity to apply advanced modeling of outcomes. Linear mixed-effects models offer a more nuanced understanding of these outcomes compared to traditional methods, which is crucial for enhancing clinical decision-making and patient care.</p>
<p><strong>How is it solved? – methods</strong></p>
<p>The authors used a retrospective dataset containing serial echocardiographic measurements from patients who underwent surgical pulmonary valve replacement at Erasmus MC between 1986 and 2017. The paper discusses the construction of the model, including dealing with missing values, correlated variables, and multicollinearity. It also covers model specification, variable selection, addressing nonlinearity, and interpretation of results. An R script is provided for implementing the model.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>The paper illustrates the construction of the model, including essential aspects such as theories of linear mixed-effects models, missing values, collinearity, interaction, nonlinearity, model specification, and results interpretation. It shows that linear mixed-effects models provide a more detailed view of repeated measurements and give more valid estimates compared to linear regression models, especially in the context of cardio-thoracic surgery outcomes research. Limitations related to model assumptions, such as linearity and normal distribution of residuals, are addressed through transformations and statistical tests.</p>
</section>
<section id="generalized-linear-mixed-models-a-practical-guide-for-ecology-and-evolution---sciencedirect" class="level2">
<h2 class="anchored" data-anchor-id="generalized-linear-mixed-models-a-practical-guide-for-ecology-and-evolution---sciencedirect"><a href="https://www.sciencedirect.com/science/article/abs/pii/S0169534709000196">Generalized linear mixed models: a practical guide for ecology and evolution - ScienceDirect</a></h2>
<p>What is the goal of the paper?</p>
<p>The goal of this paper is to provide a comprehensive guide on the use and application of Generalized Linear Mixed Models (GLMMs) for ecologists and evolutionary biologists dealing with nonnormal data types, such as counts or proportions, which often do not fit well with classical statistical procedures. The paper aims to clarify the use of GLMMs, given the popularity of these models in recent years.</p>
<p>Why is it important?</p>
<p>The importance of this paper lies in its attempt to introduce GLMMs for biologists, where data often fall outside the scope of methods taught in introductory statistics classes. The paper highlights the limitations of traditional shortcuts like data transformation or ignoring random effects and advocates for GLMMs as a more appropriate statistical approach for nonnormal data with random effects.</p>
<p>How is it solved? – methods</p>
<p>The paper reviews the use and misuse of GLMMs in biology, discusses estimation and inference, and summarizes best-practice data analysis procedures. It emphasizes the need for researchers to match their statistical approaches to their data, rather than forcing data into classical statistical frameworks. The paper discusses various estimation algorithms for fitting GLMMs, including maximum likelihood (ML), pseudo- and penalized quasilikelihood (PQL), Laplace approximations, Gauss-Hermite quadrature (GHQ), and Markov chain Monte Carlo (MCMC) algorithms.</p>
<p>Results/limitations, if any.</p>
<p>While the paper provides a broad overview of GLMM procedures and best practices, it also acknowledges the challenges and controversies in statistical issues such as null hypothesis testing, stepwise regression, and the use of Bayesian statistics. It highlights that GLMMs are powerful tools but can be challenging to use, even for statisticians, due to computational difficulties in estimating parameters, especially for complex models or large numbers of random effects.</p>
</section>
<section id="frontiers-linear-mixed-effects-models-for-within-participant-psychology-experiments-an-introductory-tutorial-and-free-graphical-user-interface-lmmgui-frontiersin.org" class="level2">
<h2 class="anchored" data-anchor-id="frontiers-linear-mixed-effects-models-for-within-participant-psychology-experiments-an-introductory-tutorial-and-free-graphical-user-interface-lmmgui-frontiersin.org"><a href="https://www.frontiersin.org/articles/10.3389/fpsyg.2015.00002/full">Frontiers | Linear mixed-effects models for within-participant psychology experiments: an introductory tutorial and free, graphical user interface (LMMgui) (frontiersin.org)</a></h2>
<p>What is the goal of the paper?</p>
<p>The goal of this paper is to introduce linear mixed-effects models (LMMs) as a versatile tool for analyzing data from within-participant psychology experiments. It seeks to address the limitations of traditional analysis methods like ANOVA in handling complex data structures, such as those involving repeated measures or nested designs. The paper also introduces LMMgui, a free, graphical user interface designed to facilitate the use of LMMs for researchers using R.</p>
<p>Why is it important?</p>
<p>The importance of this work lies in its potential to enhance the analysis of experimental psychology data by providing a more flexible and robust statistical tool that can handle the complexities of within-participant designs, such as pseudoreplication and missing data. By offering a user-friendly interface for LMM analysis, the paper aims to make advanced statistical methods more accessible to researchers, thereby improving the quality and interpretability of psychological research.</p>
<p>How is it solved? – methods</p>
<p>The paper discusses the theoretical foundation of LMMs, explaining how they can accommodate various data structures and assumptions that are commonly encountered in psychology experiments. It contrasts LMMs with traditional repeated-measures ANOVA, highlighting the advantages of LMMs in terms of their flexibility and fewer stringent assumptions. The introduction of LMMgui is a significant methodological contribution, providing a step-by-step guide on how to use this tool to specify and compare different LMMs for data analysis.</p>
<p>Results/limitations, if any.</p>
<p>While the paper primarily serves as a tutorial and does not present results from a specific study, it effectively demonstrates the application of LMMs through hypothetical examples. These examples illustrate how LMMs can be used to analyze data from within-participant designs, accounting for random effects and complex variance-covariance structures. The paper acknowledges the challenges in interpreting LMM results and the potential for increased Type I error rates in certain conditions, emphasizing the need for careful model comparison and validation.</p>
</section>
<section id="a-linear-mixed-model-to-estimate-covid-19-induced-excess-mortality---pubmed-nih.gov" class="level2">
<h2 class="anchored" data-anchor-id="a-linear-mixed-model-to-estimate-covid-19-induced-excess-mortality---pubmed-nih.gov"><a href="https://pubmed.ncbi.nlm.nih.gov/34694627/">A linear mixed model to estimate COVID-19-induced excess mortality - PubMed (nih.gov)</a></h2>
<p>What is the goal of the paper?</p>
<p>The goal of this paper is to estimate baseline mortality (mortality under non-pandemic conditions for Belgium and the Netherlands using a linear mixed model (LMM), which can account for both fixed and random effects. If baseline mortality can be modeled, then excess mortality (the measure of the increase in mortality from all causes during a specific time period) can be used to evaluate the impact of COVID-19 on mortality.</p>
<p>Why is it important?</p>
<p>Historically, 5-year weekly averages have been used to determine baseline mortality. However, this excludes year-specific trends in mortality and the effects of historical excess mortality (ex: past influenza breakouts or heat waves). Using a LMM is important because it allows for more accurate modeling that accounts for these factors in the form of random effects.</p>
<p>How is it solved? – methods</p>
<p>The paper proposes a general linear mixed model to model weekly mortality as Y<sub>tj</sub>, with t = 1,…,52 weeks and by year j = 2009…,2020.</p>
<p>The model is then adjusted to: model the cyclic pattern from year to year via random effects of Fourier terms, and reduce the influence of historical excess mortality (as mentioned above) by downweighing the residuals.</p>
<p>Results/limitations, if any.</p>
<p>Several statistics were used to evaluate the model’s forecasting accuracy, including the likelihood ratio test (LRT) and the root mean square error % (RMSE%). The models were fitted to historical mortality year from 2009- week 10 of 2020. The remaining 42 weeks of 2020 were forecasted using the LMM, along with the 5-year average model, and the ground truth data. The models all performed well, so an overall recommendation to include the down-weight procedure for past excess mortality and to include a serial correlation structure were made. The LMM did fit the mortality data better and two years were better predicted compared to the 5-year weekly average models. Many limitations exist, including differences in the reporting of COVID-19 deaths in Belgium and the Netherlands, and across the world. Additionally, it is unclear if the added complexity of LMMs provide a significant benefit over 5-year weekly average models in years besides 2014 and 2016.</p>
</section>
<section id="when-a-joint-model-should-be-preferred-over-a-linear-mixed-model-for-analysis-of-longitudinal-health-related-quality-of-life-data-in-cancer-clinical-trials---pubmed-nih.gov" class="level2">
<h2 class="anchored" data-anchor-id="when-a-joint-model-should-be-preferred-over-a-linear-mixed-model-for-analysis-of-longitudinal-health-related-quality-of-life-data-in-cancer-clinical-trials---pubmed-nih.gov"><a href="https://pubmed.ncbi.nlm.nih.gov/36765307/">When a joint model should be preferred over a linear mixed model for analysis of longitudinal health-related quality of life data in cancer clinical trials - PubMed (nih.gov)</a></h2>
<p>What is the goal of the paper?</p>
<p>The goal of this paper is to address a potential shortcoming of LMMs, which is when data is not missing at random, but instead, data is missing that is dependent on the health-related quality-of-life of the patient (i.e., data is missing because quality-of-life – the outcome - decreased). Viewing missing data like this, a survival model may be more appropriate. Or, as this paper suggests, a joint model (JM) that includes both LMM and survival sub-models.</p>
<p>Why is it important?</p>
<p>This concept is important because clinical trials are frequently employing LMMs to evaluate longitudinal data. Clinical trials involve human subjects, which are known to be more variable compared to benchtop studies. This includes variation in the completeness of patient reported outcome measures (PROMs), especially for longitudinal studies. Many times, this variation is random, and independent of the outcome measure. However, if the outcome measure itself is having an effect on the completeness, then it could represent a major bias limitation of LMMs that should be addressed using the proposed joint models.</p>
<p>How is it solved? – methods</p>
<p>This paper first introduces the LMM that is traditional in longitudinal clinical studies like this. Then, to account for the fact that observations in quality-of-life scores ends with a dropout event, a joint model is created by linking the LMM to a survival model with shared parameters.</p>
<p>This JM was then evaluated using historical clinical trial data where a standard LMM was utilized, and in several simulation studies where extreme examples of this dependence was introduced.</p>
<p>Results/limitations, if any.</p>
<p>This paper showed that poor quality-of-life scores are associated with drop out. Therefore, LMMs should be avoided when analyzing this data in clinical trials. The LMMs in both the historical studies and in the simulations were overly optimistic in estimating the quality-of-life scores. Specifically, the LMM overestimates the slope governing the prediction trajectory in both treatment and control arms. The LMM is also more optimistic for the control arm than the experimental arm, despite the protective effect of treatment on the quality-of-life score.</p>
</section>
</section>
<section id="jacobs-literature-review" class="level1">
<h1>Jacob’s Literature Review</h1>
<section id="multilevel-analysis-quantifies-variation-in-the-experimental-effect-while-optimizing-power-and-preventing-false-positives" class="level2">
<h2 class="anchored" data-anchor-id="multilevel-analysis-quantifies-variation-in-the-experimental-effect-while-optimizing-power-and-preventing-false-positives"><a href="https://bmcneurosci.biomedcentral.com/articles/10.1186/s12868-015-0228-5">Multilevel analysis quantifies variation in the experimental effect while optimizing power and preventing false positives</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The goal of this is to show how to handle nested data in neuroscience. Oftentimes data will be collected from the same sample with different experimental conditions. The paper states that this often goes over look in neuroscience</p>
<p><strong>Why is it important?</strong></p>
<p>Not only are several assumptions violated but experiential effects could be miscalculated leading to representing incorrect results.</p>
<p><strong>How is it solved? – methods</strong></p>
<p>The paper does two simulation studies to show the significance of using the appropriate significant method. Design A had cluster data that may have random effects for just the intercept.. Design B had cluster data that may have random effects in both the intercepts and experiment effect. Both cases resulted in an increase in false positive rates.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>This data is simulated and only within the context of neuroscience data. It would be beneficial to see these results to a real world dataset</p>
</section>
<section id="multimodel-inference-in-ecology-and-evolution-challenges-and-solutionsnih.gov" class="level2">
<h2 class="anchored" data-anchor-id="multimodel-inference-in-ecology-and-evolution-challenges-and-solutionsnih.gov"><a href="https://onlinelibrary.wiley.com/doi/full/10.1111/j.1420-9101.2010.02210.x">Multimodel inference in ecology and evolution: challenges and solutions(nih.gov)</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The goal of the paper is to highlight obstacles when model averaging and using information theoretic framework and their potential solutions if they exist</p>
<p><strong>Why is it important?</strong></p>
<p>A large number of ecologists and biologists are analyzing data with the Information theoretic framework rather than the traditional hypothesis testing. Modeling averaging becomes increasingly difficult with Linear Mixed models due to the fixed and random effects.</p>
<p><strong>How is it solved? – methods</strong></p>
<p>The research suggests that researchers define appropriate inputs and predictor variables and to handle collinearity with extreme care especially when dealing with the random effects of LMMs.The paper then goes on to propose strategies on model averaging and definition top model sets.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>As the paper mentions, it is NOT an exhaustive survival of the potential problems of applying model averaging under an IT framework. Some problems still exist like determining which IT criteria to use when comparing models with random factors.</p>
</section>
<section id="linear-mixed-model-for-analyzing-longitudinal-data-a-simulation-study-of-children-growth-differences-sciencedirect.com" class="level2">
<h2 class="anchored" data-anchor-id="linear-mixed-model-for-analyzing-longitudinal-data-a-simulation-study-of-children-growth-differences-sciencedirect.com"><a href="https://www.sciencedirect.com/science/article/pii/S187705091732121X">Linear Mixed Model for Analyzing Longitudinal Data: A Simulation Study of Children Growth Differences (sciencedirect.com)</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The goal of the paper is to use Linear Mixed Models to analyze multilevel data of developmental growth rates in children. Different covariance structures were modeled within the LMM to capture correlated data through time.</p>
<p><strong>Why is it important?</strong></p>
<p>Growth curves in children are usually represented as a 2 level data structure. At level 2 is the individual child and at the second level is each individual observation. Traditional linear models are not effective due to the non-independence of the data.</p>
<p><strong>How is it solved? – methods</strong></p>
<p>The simulation study found that the UN covariance performed the best although it suffered from efficiency because of the high number of parameters leaving the ARH(1) as a solid alternative. The data was also not naturally collected but simulated</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>The simulation study found that the UN covariance performed the best although it suffered from efficiency because of the high number of parameters leaving the ARH(1) as a solid alternative. The data was also not naturally collected but simulated</p>
</section>
<section id="a-protocol-for-conducting-and-presenting-results-of-regression-type-analyses" class="level2">
<h2 class="anchored" data-anchor-id="a-protocol-for-conducting-and-presenting-results-of-regression-type-analyses"><a href="https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.12577">A protocol for conducting and presenting results of regression-type analyses</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The goal of the paper is to streamline analysis by giving the reader a 10 step protocol. It helps fellow researchers select models, justify assumptions, and validate models.</p>
<p><strong>Why is it important?</strong></p>
<p>This paper is great for researchers new to Linear Mixed models and are looking to use its advantages on a dataset. The protocol is extremely straightforward. Linear mixed models offer more in depth analysis and it is important that all researchers know it to further the field of ecology.</p>
<p><strong>How is it solved? – methods</strong></p>
<p>The paper has 10 steps with very concrete examples that include sample datasets, visualizations, and results. The reader has something tangible and can directly apply what they learned to another dataset.</p>
<p><strong>Results/limitations, if any</strong></p>
<p>The paper is limited by showing sample results and shows no empirical data. It identifies potential pitfalls like overdispersion of fitted models but offers no real solution</p>
</section>
<section id="level-logical-explanations-visualizations-of-estimates-in-linear-mixed-models-recommendations-for-reporting-multilevel-data-and-analyses" class="level2">
<h2 class="anchored" data-anchor-id="level-logical-explanations-visualizations-of-estimates-in-linear-mixed-models-recommendations-for-reporting-multilevel-data-and-analyses"><a href="https://bmcmedresmethodol.biomedcentral.com/articles/10.1186/s12874-019-0876-8">LEVEL (Logical Explanations &amp; Visualizations of Estimates in Linear mixed models): recommendations for reporting multilevel data and analyses</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>Researchers use LLMs to study hierarchical data and often report them under different names like mixed effects models, multilevel data, contextual analysis and hierarchical studies. There is no standardization across these papers for analyzing hierarchical data which leads to different aspects being reported. The goal of the paper is to make a standardized process for analyzing multilevel data</p>
<p><strong>Why is it important?</strong></p>
<p>This is important so studies across different time periods can be compared more easily.</p>
<p><strong>How is it solved? – methods</strong></p>
<p>The paper suggests using the LEVEL (Logical Explanations &amp; Visualizations of Estimates in Linear mixed models) as framework for conducting studies with reporting recommendations</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>Lack of flexibility. Sticking to a framework can inhibit creative analysis since you’re always looking at the framework for guidance.</p>
</section>
<section id="estimation-and-selection-in-linear-mixed-models-with-missing-data-under-compound-symmetric-structure-nih.gov" class="level2">
<h2 class="anchored" data-anchor-id="estimation-and-selection-in-linear-mixed-models-with-missing-data-under-compound-symmetric-structure-nih.gov"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9621253/">Estimation and selection in linear mixed models with missing data under compound symmetric structure (nih.gov)</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>Missing values occur all the time in real data. Statisticians and scientists use linear mixed models as a way to circumvent this issue. This paper aims to examine the estimation and model selection performance when faced with different rates of missing data. The paper employs two types of missing data. Missing at random and not at random.</p>
<p><strong>Why is it important?</strong></p>
<p>Given the frequency of missing data it’s important to know the impact it has on the model’s results. It’s also important to be able to</p>
<p><strong>How is it solved? – methods</strong></p>
<p>Missingness of data is recorded using an indicator based matrix and then a likelihood based estimator is made to capture the probability of distribution of the observed data given the model parameters.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>There is adequate model performance when there is a moderate amount of missingness in the data. However, the paper focuses on compound symmetric structures which assumes equal variance among any given pair of observations.</p>
</section>
</section>
<section id="syeds-literature-review" class="level1">
<h1>Syed’s Literature Review</h1>
<section id="an-introduction-to-linear-mixed-effects-modeling-in-r" class="level2">
<h2 class="anchored" data-anchor-id="an-introduction-to-linear-mixed-effects-modeling-in-r"><a href="https://journals.sagepub.com/doi/10.1177/2515245920960351">An Introduction to Linear Mixed-Effects Modeling in R</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The goal of the tutorial is to provide both theoretical understanding and practical guidance on implementing mixed-effects models in R, particularly for researchers with basic statistical knowledge but limited experience in using these models. It aims to address the limitations of traditional statistical methods like repeated measures ANOVAs in analyzing correlated data and to introduce mixed-effects modeling as a more flexible and appropriate approach.</p>
<p><strong>Why is it important?</strong></p>
<p>Understanding mixed-effects modeling is crucial for researchers, especially in fields like experimental psychology where traditional methods may not adequately address the complexities of correlated data. By offering accessible explanations and practical examples, the tutorial aims to empower researchers to effectively analyze their data using mixed-effects models, thereby improving the quality and validity of their research findings.</p>
<p><strong>How is it solved? – methods</strong></p>
<p>The tutorial provides a theoretical introduction to mixed-effects modeling, explaining concepts such as fixed and random effects in simple terms. It contrasts mixed-effects modeling with traditional methods like repeated measures ANOVAs, highlighting the advantages of the former in handling correlated data and various types of response variables. Practical guidance is offered through R code snippets and example data, allowing readers to follow along and implement mixed-effects models in their own research.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>The tutorial does not present empirical results but rather serves as an educational resource. It effectively communicates the benefits of mixed-effects modeling and provides step-by-step instructions for implementation in R.</p>
</section>
<section id="a-brief-introduction-to-mixed-effects-modelling-and-multi-model-inference-in-ecology" class="level2">
<h2 class="anchored" data-anchor-id="a-brief-introduction-to-mixed-effects-modelling-and-multi-model-inference-in-ecology"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5970551/">A brief introduction to mixed effects modelling and multi-model inference in ecology</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The paper aims to provide best practices for applying linear mixed effects models (LMMs) to biological data, particularly in ecology and evolutionary studies. It seeks to address the complexities of ecological data and offer guidance on model selection, interpretation, and common pitfalls encountered during modeling.</p>
<p><strong>Why is it important?</strong></p>
<p>With the increasing use of LMMs in biological data analysis, particularly in ecology, establishing best practices is critical for enhancing the robustness of conclusions drawn from ecological and evolutionary studies. Effective application of LMMs can improve the accuracy and reliability of research findings.</p>
<p><strong>How is it solved? – methods</strong> The paper discusses various aspects of applying LMMs to biological data, including model selection, error structure, data transformation, and methods for model selection. It emphasizes the importance of careful consideration and consultation with a statistician, particularly in complex situations.</p>
<p><strong>Results/limitations, if any</strong></p>
<p>While the paper does not present empirical results, it offers practical solutions and recommendations for researchers working with ecological data. It effectively communicates the advantages and disadvantages of LMMs and provides valuable insights for their application in ecology and evolutionary studies.</p>
</section>
<section id="robustness-of-linear-mixed-effects-models-to-violations-of-distributional-assumptions" class="level2">
<h2 class="anchored" data-anchor-id="robustness-of-linear-mixed-effects-models-to-violations-of-distributional-assumptions"><a href="https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.13434">Robustness of linear mixed-effects models to violations of distributional assumptions</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The paper aims to investigate the robustness of linear mixed-effects models (LMMs) in analyzing complex datasets commonly found in ecology and evolution. It evaluates the impact of various violations of distributional assumptions and missing random effect components on model estimates.</p>
<p><strong>Why is it important?</strong></p>
<p>Understanding the robustness of LMMs is crucial for researchers working with complex ecological and evolutionary datasets. Despite potential violations of assumptions and missing components, LMMs are widely used in these fields. Assessing their robustness helps ensure the accuracy and reliability of model estimates, even in challenging scenarios.</p>
<p><strong>How is it solved? – methods</strong></p>
<p>The study evaluates the impact of skewed, bimodal, and heteroscedastic random effect and residual variances, as well as the effects of missing random effect terms and correlated fixed effect predictors on model estimates. It likely employs simulations or analytical approaches to systematically assess the performance of LMMs under various conditions.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>The results indicate that while violations of assumptions may lead to slight biases and decreased precision in estimates, the overall robustness of LMMs allows for accurate and unbiased estimation of fixed and random effects.</p>
</section>
<section id="to-transform-or-not-to-transform-using-generalized-linear-mixed-models-to-analyse-reaction-time-data" class="level2">
<h2 class="anchored" data-anchor-id="to-transform-or-not-to-transform-using-generalized-linear-mixed-models-to-analyse-reaction-time-data"><a href="https://www.frontiersin.org/journals/psychology/articles/10.3389/fpsyg.2015.01171/full">To transform or not to transform: using generalized linear mixed models to analyse reaction time data</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The paper aims to challenge the common practice of transforming reaction time (RT) data to meet normality assumptions in statistical analyses within cognitive psychology research. It proposes generalized linear mixed-effect models (GLMMs) as a solution to accurately analyze RT data without the need for transformation, thus avoiding potential theoretical implications and misleading conclusions.</p>
<p><strong>Why is it important?</strong></p>
<p>It highlights the discrepancy between analyses of raw RT data and transformed RT data, as demonstrated by Balota et al.&nbsp;(2013), emphasizing the need for a more nuanced approach to analyzing RT data. By advocating for GLMMs, the paper aims to promote proper assessment of individual differences and enhance the testing of cognitive theories.</p>
<p><strong>How is it solved? – Methods</strong></p>
<p>The study discusses the theoretical decisions involved in specifying a GLMM and provides reanalysis of datasets from Balota et al.&nbsp;(2013) to illustrate the application of GLMMs in RT data analysis. It emphasizes the importance of analyzing changes in RT distribution at a finer level to capture more accurate measures of group performance and effectively test cognitive theories.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>The paper suggests that GLMMs offer a more robust approach to analyzing RT data compared to traditional methods like linear mixed-effect models (LMMs) with transformed data. However, it acknowledges the complexities of addressing skewed dependent variables like RT in LMMs and the potential challenges in adopting GLMMs, such as the need for careful model specification.</p>
</section>
<section id="analysing-disease-incidence-data-from-designed-experiments-by-generalized-linear-mixed-models" class="level2">
<h2 class="anchored" data-anchor-id="analysing-disease-incidence-data-from-designed-experiments-by-generalized-linear-mixed-models"><a href="https://bsppjournals-onlinelibrary-wiley-com.ezproxy.lib.uwf.edu/doi/abs/10.1046/j.1365-3059.1999.00383.x">Analysing disease incidence data from designed experiments by generalized linear mixed models</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The paper aims to introduce generalized linear mixed models (GLMMs) as a robust method for analyzing disease incidence data from designed experiments, specifically addressing overdispersion issues common in epidemiological research.</p>
<p><strong>Why is it important?</strong></p>
<p>It highlights the inadequacy of traditional methods like ANOVA for such data, underlining the need for alternative approaches like GLMMs to better capture the complexities of disease clustering and aggregation.</p>
<p><strong>How is it solved? – Methods</strong></p>
<p>The study presents GLMMs as a versatile tool, capable of accommodating both fixed and random effects, thus offering a more flexible framework for analyzing disease incidence data. It illustrates the application of GLMMs using real-world data from an experiment on downy mildew incidence in grapevines.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>The analysis using GLMMs reveals significant treatment effects and provides parameter estimates. However, the approach is not without limitations, including assumptions made in modeling and the potential challenge of interpreting results accurately, particularly in complex experimental designs.</p>
</section>
</section>
<section id="saras-literature-review" class="level1">
<h1>Sara’s Literature Review</h1>
<section id="model-selection-in-linear-mixed-effect-models" class="level2">
<h2 class="anchored" data-anchor-id="model-selection-in-linear-mixed-effect-models"><a href="https://www.sciencedirect.com/science/article/pii/S0047259X12000395">Model selection in linear mixed effect models</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The goal of the paper is to improve variable selection and parameter estimation in linear mixed effect models, which are critical for analyzing longitudinal, panel, and cross-sectional data in various scientific domains.</p>
<p><strong>Why is it important?</strong></p>
<p>Improving variable selection and parameter estimation is crucial because it directly impacts the accuracy and reliability of data analysis across scientific fields. Efficiently identifying relevant variables and accurately estimating their effects are essential for drawing valid conclusions from complex data structures.</p>
<p><strong>How is it solved? – Methods</strong></p>
<p>The authors introduce a simple, iterative procedure that employs the smoothly clipped absolute deviation (SCAD) penalty function to estimate and select both fixed and random effects in these models. This approach is highlighted for being a consistent variable selection method with some oracle properties, suggesting it can perform almost as well as if the true underlying model were known.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>The approach’s effectiveness and efficiency are validated through simulation studies and real data analysis. Nevertheless, the paper also points out limitations, including the method’s dependence on certain conditions for its asymptotic properties to hold and the potential computational challenges encountered with high-dimensional datasets.</p>
</section>
<section id="random-effects-structure-for-testing-interactions-in-linear-mixed-effects-models" class="level2">
<h2 class="anchored" data-anchor-id="random-effects-structure-for-testing-interactions-in-linear-mixed-effects-models"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3672519/">Random Effects Structure for Testing Interactions in Linear Mixed-Effects Models</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The goal is to provide a more accurate method for testing interactions within linear mixed-effects models, critiquing existing guidelines and proposing new ones that emphasize the inclusion of random slopes for the highest-order combination of within-unit factors in interactions.</p>
<p><strong>Why is it important?</strong></p>
<p>This is important because accurately testing interactions in mixed-effects models is crucial for statistical analyses, especially in avoiding high Type I error rates. The paper aims to refine the approach to these models to ensure more reliable results.</p>
<p><strong>How is it solved? – Methods</strong></p>
<p>The author employs Monte Carlo simulations to test the proposed guidelines, demonstrating that neglecting critical random slopes can significantly increase the chance of a false rejection of the null hypothesis. Including appropriate random slopes in the model is shown to ensure better performance.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>The findings highlight that including the correct random slopes in mixed-effects models greatly improves model performance, particularly in accurately testing interactions between categorical variables. However, the paper’s limitations include its focus on interactions between categorical variables and the specific conditions of the simulations used.</p>
</section>
<section id="pymer4-connecting-r-and-python-for-linear-mixed-modeling" class="level2">
<h2 class="anchored" data-anchor-id="pymer4-connecting-r-and-python-for-linear-mixed-modeling"><a href="https://joss.theoj.org/papers/10.21105/joss.00862">Pymer4: Connecting R and Python for Linear Mixed Modeling</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The goal is to develop Pymer4, a tool that bridges R and Python for linear mixed modeling, addressing the gap in Python for a package as flexible as R’s lme4 for complex data analysis.</p>
<p><strong>Why is it important?</strong></p>
<p>Pymer4 is significant for providing Python users with an accessible, integrated tool for linear mixed modeling, which was previously lacking, enhancing the analytical capabilities within the Python ecosystem.</p>
<p><strong>How is it solved? – Methods</strong></p>
<p>Pymer4 offers a solution by by leveraging the rpy2 library, as it connects to R’s lme4 package, offering a Pythonic interface for mixed modeling that integrates well with scientific Python tools, simplifying the analysis process.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>Pymer4 successfully extends lme4’s functionality to Python users, offering features like significance testing and data visualization integration, enhancing multilevel model analysis. The paper focuses on the tool’s capabilities without detailing specific limitations.</p>
</section>
<section id="a-powerful-and-flexible-linear-mixed-model-framework-for-the-analysis-of-relative-quantification-rt-pcr-data" class="level2">
<h2 class="anchored" data-anchor-id="a-powerful-and-flexible-linear-mixed-model-framework-for-the-analysis-of-relative-quantification-rt-pcr-data"><a href="https://pubmed.ncbi.nlm.nih.gov/19422910/">A powerful and flexible linear mixed model framework for the analysis of relative quantification RT-PCR data</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The paper introduces a novel linear mixed model framework for analyzing relative quantification RT-PCR data, aiming to overcome the limitations of existing statistical methods by providing more accurate and flexible analysis tools.</p>
<p><strong>Why is it important?</strong></p>
<p>This framework is crucial for its potential to enhance the statistical power and flexibility in analyzing RT-PCR data, enabling researchers to conduct more reliable and varied analyses of gene expression across different experimental conditions.</p>
<p><strong>How is it solved? – Methods</strong></p>
<p>The method involves a sophisticated statistical approach that incorporates both fixed and random effects in a linear mixed model, allowing for a more nuanced analysis of RT-PCR data that accounts for various sources of variability.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>The framework has been shown to yield more accurate and statistically powerful results compared to traditional methods, facilitating better decision-making in biological research. The paper thoroughly evaluates the model’s performance and discusses its applicability to a wide range of experimental designs.</p>
</section>
<section id="using-generalized-linear-mixed-models-to-evaluate-inconsistency-within-a-network-meta-analysis" class="level2">
<h2 class="anchored" data-anchor-id="using-generalized-linear-mixed-models-to-evaluate-inconsistency-within-a-network-meta-analysis"><a href="https://www.sciencedirect.com/science/article/pii/S1098301515050706">Using Generalized Linear Mixed Models to Evaluate Inconsistency within a Network Meta-Analysis</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The goal of the paper is to demonstrate how generalized linear mixed models (GLMMs) can evaluate inconsistency within network meta-analyses, improving upon traditional models by using an arm-based approach for more accurate results.</p>
<p><strong>Why is it important?</strong></p>
<p>The paper addresses the challenge of inconsistency between direct and indirect evidence in network meta-analysis, which can compromise the validity of conclusions, offering a more reliable framework for analysis.</p>
<p><strong>How is it solved? – Methods</strong></p>
<p>The authors propose an arm-based GLMM approach, which allows for flexible modeling of different outcome variables and shows improved accuracy over contrast-based methods, especially when event rates are low.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>The arm-based model provided more accurate evaluations of design inconsistency and treatment effects compared to traditional contrast-based approaches, highlighting its utility in complex analyses involving many treatments and designs.</p>
</section>
<section id="generalized-linear-mixed-model-glmm-trees-a-flexible-decision-tree-method-for-multilevel-and-longitudinal-data" class="level2">
<h2 class="anchored" data-anchor-id="generalized-linear-mixed-model-glmm-trees-a-flexible-decision-tree-method-for-multilevel-and-longitudinal-data"><a href="https://www.tandfonline.com/doi/full/10.1080/10503307.2020.1785037">Generalized linear mixed-model (GLMM) trees: A flexible decision-tree method for multilevel and longitudinal data</a></h2>
<p><strong>What is the goal of the paper?</strong></p>
<p>The goal of the paper is to introduce GLMM trees, a method combining generalized linear mixed models (GLMMs) with decision trees for analyzing multilevel and longitudinal data, offering a novel approach to clinical prediction problems.</p>
<p><strong>Why is it important?</strong></p>
<p>GLMM trees provide an interpretable and flexible method that can handle complex data structures, improving upon traditional models by simplifying the analysis and enhancing the understanding of data patterns.</p>
<p><strong>How is it solved? – Methods</strong></p>
<p>The paper employs GLMM trees to analyze a large dataset from UK mental health services, comparing its performance with traditional GLMMs and random forests to demonstrate its predictive accuracy and efficiency.</p>
<p><strong>Results/limitations, if any.</strong></p>
<p>The method achieves similar predictive accuracy to traditional GLMMs and random forests but with fewer variables, showcasing its potential to streamline clinical decision-making.</p>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = false;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>